src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityWidgetV2.java

```diff
@@ -335,7 +335,10 @@ private void configureActions() {
           "There is no data to edit"
         );
     }
-    this.actionMenu.setActionVisible(Action.EDIT_TABLE_DATA, true);
+    this.actionMenu.setActionVisible(
```

Comment:
Bug where the EDIT_TABLE_DATA action should not be visible on non-editable table types (datasets, materialized views)

Reply:
https://sagebionetworks.jira.com/browse/SWC-6352